core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -305,6 +308,13 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     case RescheduleJob =>
       freePool = freePool - sender()
       busyPool = busyPool - sender()
+    case message: ByteSizeMessage =>
+      logging.info(
+        this,
+        s"user memory is reconfigured from ${latestUserMemory.toString} to ${message.userMemory.toString}")
+      latestUserMemory = message.userMemory
+    case UserMemoryQuery =>
+      sender() ! latestUserMemory.toString
```

Comment:
How about sending the intact obejct directly and handle the type conversion in the InvokerServer layer?
I think this kind of type conversion is only related to the InvokerServer and the client.

Reply:
hm..
When we want to change invoker's user memory, in this pr, need to send the request to controller, there has one benefit  that can modfiy many invoker's user memory with one request only. e.g.
```
curl  -u admin:admin -X POST http://xxx.xxx.xxx.xxx:10001/config/memory  -d '
[
{"invoker":0,"memory": "20480 MB"},
{"invoker":1,"memory": "10240 MB"}
{"invoker":2,"memory": "51200 MB"}
]
'
```
if send `change invoker user memory` request to invokerServer, if want to modify many invokers's user memory, need to send http request many times.

Reply:
Ok since this is just a `ByteSize` I think we can take this approach.